waku/v2/protocol/filter/filter_subscribers.go

```diff
@@ -15,6 +15,13 @@ type Subscriber struct {
 }
 
 func (sub Subscriber) HasContentTopic(topic string) bool {
+<<<<<<< HEAD
```

Comment:
Looks like we have some merge conflicts in here

Reply:
yup, fixed. I keep forgetting to save the file, after I resolve the conflicts 🤦